services/terrain/src/terrain/clients/data_info/raw.clj

```diff
@@ -200,20 +201,25 @@
 (defn set-avus
   "Set the iRODs AVUs to a specific set."
   [user path-uuid avu-map]
-  (request :put ["data" path-uuid "avus"]
-           (mk-req-map user (json/encode {:irods-avus avu-map}))))
+  (request :post ["data" path-uuid "avus"]
+           (mk-req-map user (json/encode avu-map))))
 
 (defn add-avus
   "Add AVUs to a data item."
   [user path-uuid avu-map]
-  (request :post ["data" path-uuid "avus"]
-           (mk-req-map user (json/encode {:irods-avus avu-map}))))
+  (request :put ["data" path-uuid "avus"]
+           (mk-req-map user (json/encode avu-map))))
+
+(defn metadata-copy
+  [user force path-uuid copy-request]
+  (request :post ["data" path-uuid "avus" "copy"]
+           (mk-req-map user (json/encode copy-request) {:force force})))
 
 (defn admin-add-avus
   "Add AVUs, allowing administrative AVUs to be included, for a data item."
   [user path-uuid avu-map]
-  (request :post ["admin" "data" path-uuid "avus"]
-           (mk-req-map user (json/encode {:irods-avus avu-map}))))
+  (request :put ["admin" "data" path-uuid "avus"]
+           (mk-req-map user (json/encode avu-map))))
```

Comment:
Is this right? PUT should be a 'set' operation (replacing the resource) vs. POST which can be an addition (modifying the resource). This looks backwards to me.